DataSetExplorer/Database/DataSetExplorerContext.cs

```diff
@@ -0,0 +1,51 @@
+﻿using DataSetExplorer.DataSetBuilder.Model;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DataSetExplorer.Database
+{
+    public class DataSetExplorerContext : DbContext
+    {
+        public DataSetExplorerContext(DbContextOptions<DataSetExplorerContext> options) : base(options)
+        {
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Annotator>().ToTable("Annotators");
```

Comment:
I don't think you need most of this code, if any.

![image](https://user-images.githubusercontent.com/7092212/126432452-b68dcf03-15db-405f-b089-a057f2569ebf.png)
Looking at the image above (from the SmartTutor DBContext), we define the fields of the DBContext so that we can use them to work with EFCore and the database. Simply by defining the fields we will already get a table when running using EF to generate our schema, as well as the appropriate foreign keys.

The reason why there is some configuration in our SmartTutor DBContext is mostly because of inheritance (e.g., the Images inherit the LearningObject class and should be placed in a separate table).

Try removing all the code from here, adding the appropriate fields and running add-migration on a new DB to see the results.